Description: Debianize
 Make this XStatic package to use what's shipped by libjs-rickshaw, instead of
 the embedded files.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-08-17

--- python-xstatic-rickshaw-1.5.0.0.orig/xstatic/pkg/rickshaw/__init__.py
+++ python-xstatic-rickshaw-1.5.0.0/xstatic/pkg/rickshaw/__init__.py
@@ -11,7 +11,7 @@ NAME = __name__.split('.')[-1] # package
                                # please use a all-lowercase valid python
                                # package name
 
-VERSION = '1.4.6' # version of the packaged files, please use the upstream
+VERSION = '1.5.0' # version of the packaged files, please use the upstream
                   # version number
 BUILD = '2' # our package build number, so we can release new builds
              # with fixes for xstatic stuff.
@@ -34,9 +34,9 @@ HOMEPAGE = 'http://code.shutterstock.com
 LICENSE = '(same as %s)' % DISPLAY_NAME
 
 from os.path import join, dirname
-BASE_DIR = join(dirname(__file__), 'data')
+#BASE_DIR = join(dirname(__file__), 'data')
 # linux package maintainers just can point to their file locations like this:
-#BASE_DIR = '/usr/share/javascript/rickshaw'
+BASE_DIR = '/usr/share/javascript/rickshaw'
 
 LOCATIONS = {
     # CDN locations (if no public CDN exists, use an empty dict)
